x-pack/platform/plugins/private/observability_ai_assistant_management/public/routes/components/settings_tab/product_doc_entry.tsx

```diff
@@ -19,14 +19,11 @@ import {
 } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n-react';
-import { UseKnowledgeBaseResult } from '@kbn/ai-assistant/src/hooks';
+import { UseKnowledgeBaseResult } from '@kbn/ai-assistant';
 import { getMappedInferenceId } from '../../../helpers/inference_utils';
-import { useKibana } from '../../../hooks/use_kibana';
 import { useGetProductDoc } from '../../../hooks/use_get_product_doc';
 
 export function ProductDocEntry({ knowledgeBase }: { knowledgeBase: UseKnowledgeBaseResult }) {
```

Comment:
Can we rename this component `ProductDocsSetting`. "Entry" makes it sounds like this is an actual product doc